.github/workflows/check_cve_number.yml

```
@@ -0,0 +1,27 @@
+name: Check angular.js CVEs on SNYK
+
+on:
+  schedule:
+    - cron: "0 0 1 * *"
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Check for new CVEs
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout main
+        uses: actions/checkout@v3
+      - name: Check CVEs
+        run: |
+          bash ${GITHUB_WORKSPACE}/scripts/github-actions/check-cves.sh
+      - name: Send message to Slack channel
+        if: ${{ failure() && (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/v1.3.x') }}
```

As we change the main branch to v1.3.x
```suggestion
        if: ${{ failure() }}
```